src/Xliff.js

```diff
@@ -0,0 +1,970 @@
+/*
+ * Xliff.js - model an xliff file
+ *
+ * Copyright © 2016-2017, 2019-2022 HealthTap, Inc. and JEDLSoft
```

Comment:
it's a file from the new repository. Isn't it ok to remove the old copyright year?